packages/manager/modules/anthos/src/dashboard/general-information/software-update/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,13 @@
+{
+  "anthos_tenant_dashboard_general_information_software_update_back": "Retour aux informations générales",
+  "anthos_tenant_dashboard_general_information_software_update_success": "success",
+  "anthos_tenant_dashboard_general_information_software_update_error": "error",
```

Comment:
**issue(blocking):** Missing content here (as describe in the Pull Request description).